modules/home-manager/sops.nix

```diff
@@ -124,6 +125,16 @@ in {
       '';
     };
 
+    defaultSopsKey = mkOption {
```

Comment:
Currently I don't have a better name but defaultSopsKey sounds very confusing because it's quite overloaded and we have private / public keys everywhere.

Reply:
For your usecase, do you actually need this option?
Is it not enough to set this instead?

```
{
  sops.defaultSopsFormat = "binary";
}
```